nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/state/flow/flow.effects.ts

```diff
@@ -222,6 +224,17 @@ export class FlowEffects {
                             }),
                             catchError((error) => of(FlowActions.flowApiError({ error: error.error })))
                         );
+                    case ComponentType.RemoteProcessGroup:
+                        return from(this.flowService.getParameterContexts()).pipe(
```

Comment:
The create Remote Process Group dialog does not require the Parameter Contexts. This `case` should be able to simply return the action for opening the dialog. Kind of like the cases for Processors or Ports.